metric/system/process/process_windows.go

```diff
@@ -140,11 +134,25 @@ func FillPidMetrics(_ resolve.Resolver, pid int, state ProcState, _ func(string)
 
 	state.CPU.StartTime = unixTimeMsToTime(startTime)
 
+	return state, nil
+}
+
+// FillMetricsRequiringMoreAccess
+// All calls that need more access rights than
+// windows.PROCESS_QUERY_LIMITED_INFORMATION
+func FillMetricsRequiringMoreAccess(pid int, state ProcState) (ProcState, error) {
 	argList, err := getProcArgs(pid)
 	if err != nil {
-		return state, fmt.Errorf("error fetching process args: %w", err)
+		return state, fmt.Errorf("error fetching process args: %w", NotEnoughPrivilegesErr{Err: err})
```

Comment:
If I'm reading this right, it turns all errors from `getProcArgs` into not enough privileges errors.  But if I look at `getProcArgs` it might fail for reasons other than permissions.  Can we do this for cases where it is really just a permission problem? or maybe change the name?

Reply:
You're right. On my first implementation I had something like `CanIgnoreErr` to transmit the idea that even though we got an error we can still trust and use the metrics that were returned.

I'll try to came up with a better name.

Reply:
I fixed it on 15e3d6200 and also improved the logging.